### ISSUE TITLE
prettier.formatFile not found in version 1.1.19

### ISSUE THREAD

**Issue by nicknisi** [neoclide/coc-prettier#101]
I updated to version 1.1.19 this morning and now I'm getting the following error:

```
[coc.nvim] Command: prettier.formatFile not found
```

I'm trying to run the following command, but it also seems that prettier is no longer formatting on save.

```vim
command! -nargs=0 Prettier :CocCommand prettier.formatFile
```

**Comment by michie1** [comment]
When I run `:Prettier` it shows `Command error: Invalid Version: [object Object]`. Do you have the same error?

**Comment by chemzqm** [comment]
Maybe you're using local prettier which is too old.

**Comment by tonietto** [comment]
I can confirm that upgrading prettier from 1.18.2 to 2.2.0 fixed the issue for me.

**Comment by adworacz** [comment]
Unfortunately, I need to continue to use the old version of Prettier.

I can't find any documentation on how to lock coc to using a specific extension version. How can I lock coc to use 1.1.18 of the coc-prettier extension?

**Comment by chemzqm** [comment]
> I can't find any documentation on how to lock coc to using a specific extension version

`:CocList extensions` and press `<tab>`, you can get lock action.

**Comment by adworacz** [comment]
> > I can't find any documentation on how to lock coc to using a specific extension version
> 
> `:CocList extensions` and press `<tab>`, you can get lock action.

Yeah, this only works well if the user does the `:CocInstall coc-prettier@1.x.xx` *first*, because the lock action applies to the currently installed version.

But it looks like myself and several others are still experiencing this "prettier.formatFile not found" issue even with a downgrade, which surprises me.

**Comment by adworacz** [comment]
Welp - this is embarrassing. The reason why I was getting the "prettier.formatFile not found" error even after installing an older version of the prettier extension is because the extension got disabled somehow (I think I must have done it and forgotten about it).

To fix it (after I'd installed 1.1.17 using the instructions I posted earlier (.18 works too)), I simply loaded up the prettier extension with `:CocList extensions`, selected the coc-prettier extension (which was marked with `-` to show it was disabled), pressed `Tab` on my keyboard, then `e` for enable. Restarted Coc just to be sure, and now everything's working fine.
